lib/src/main/java/com/segment/analytics/kotlin/destinations/consent/ConsentBlocker.kt

```diff
@@ -34,7 +36,7 @@ class ConsentBlockingPlugin(
             requiredConsentCategories.forEach {
                 if (!consentJsonArray.contains(it)) {
```

Comment:
maybe make `consentJsonArray` a set instead of a list. reduce the complexity of `contains` from O(n) to O(1)